R/tm_g_response.R

```diff
@@ -333,7 +336,7 @@ srv_g_response <- function(id,
       plot_call <- substitute(
         expr =
           ggplot(ANL2, aes(x = x_cl, y = ns)) +
-            geom_bar(aes(fill = resp_cl), stat = "identity", position = arg_position),
+            geom_bar(aes(fill = resp_cl), stat = "identity", position = arg_position, dd = ""),
```

Comment:
Is this right?

Reply:
i was testing with this and i forgot it there, will remove.